### PR TITLE
SBERDOMA-889 fix suggest 

### PR DESCRIPTION
**Pull request by leonid-d**
bonus: fix affix layout :)

### CODE REVIEW

File: apps/condo/domains/common/components/ActionBar.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const ActionBar: React.FC<IActionBarProps> = ({ children, fullscreen = false })
     return (
         <Affix offsetBottom={24} style={{ width: fullscreen ? '100%' : 'unset' }}>
             <div css={actionBar}>
-                <Space wrap={true} size={[40, 24]}>
+                <Space wrap={true} size={[40, 24]} style={{ marginBottom: '0 !important' }}>
```
Comment (Dimitreee) [inline apps/condo/domains/common/components/ActionBar.tsx:27]: Bad practise. We shouldnt use !important here. Use common ant'd components and props.

File: apps/condo/domains/common/components/GraphQlSearchInput.tsx [hunk 2]
```diff
@@ -65,6 +65,11 @@ export const GraphQlSearchInput: React.FC<ISearchInputProps> = (props) => {
     function handleSelect (value, option) {
         setSelected(option.children)
 
+        if (props.mode === 'multiple') {
```
Comment (Dimitreee) [inline apps/condo/domains/common/components/GraphQlSearchInput.tsx:68]: What is this fix about? Please describe your solution

**Review by Dimitreee — CHANGES_REQUESTED** [review]

File: apps/condo/domains/common/components/GraphQlSearchInput.tsx (hunk 2 above)
Comment (leonid-d) [reply apps/condo/domains/common/components/GraphQlSearchInput.tsx:68]: In shareTIcketModal, we have a multiselect for user choice, in which we need to clear the search field after selecting the user with whom we want to share the ticket.

File: apps/condo/domains/common/components/GraphQlSearchInput.tsx [hunk 3]
```diff
@@ -65,6 +65,11 @@ export const GraphQlSearchInput: React.FC<ISearchInputProps> = (props) => {
...
+            setValue('')
+            handleSearch('')
```
Comment (AntonAL) [inline apps/condo/domains/common/components/GraphQlSearchInput.tsx:70]: It triggers extra query on select item.
```suggestion
```
Can `autoClearSearchValue` property of `Select` be used in your case?
https://ant.design/components/select/
I've tried locally, it seems to be working.

**Review by AntonAL — APPROVED** [review]

**Review by Dimitreee — APPROVED** [review]

**Pull request merged**